src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsTrackingOptInRequestsOasis.java

```diff
@@ -356,51 +356,51 @@ public void adsTrackingAcceptedForComscore(List<String> urlPatterns) {
     modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
   }
 
-  @NetworkTrafficDump(useMITM = true)
-  @Execute(trackingOptIn = false)
-  @Test(
-      dataProviderClass = TrackingOptInDataProvider.class,
-      dataProvider = "adsNpaHiViDataProviderOasis",
-      groups = "AdsOptInAcceptedOasis"
-  )
-  public void adsTrackingAcceptedForHiViAdSlots(List<String> urlPatterns) {
-    networkTrafficInterceptor.startIntercepting();
-    TrackingOptInModal modal = new TrackingOptInModal();
-    modal.acceptOptInModal(driver, POLAND, ADS_HIVI_PAGE);
-    modal.scrollTo(driver.findElement(By.className("wds-global-footer__header")));
-
-    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
-  }
-
-  @NetworkTrafficDump(useMITM = true)
-  @Execute(trackingOptIn = false)
-  @Test(
-      dataProviderClass = TrackingOptInDataProvider.class,
-      dataProvider = "adsGoogleAnalyticsDataProvider",
-      groups = "AdsOptInAcceptedOasis"
-  )
-  public void adsTrackingAcceptedForGoogleAnalytics(List<String> urlPatterns) {
-    networkTrafficInterceptor.startIntercepting();
-    TrackingOptInModal modal = new TrackingOptInModal();
-    modal.acceptOptInModal(driver, POLAND, ADS_HOME_PAGE);
-
-    modal.verifyTrackingRequestsNotSend(urlPatterns, networkTrafficInterceptor);
-  }
-
-  @NetworkTrafficDump(useMITM = true)
-  @Execute(trackingOptIn = false)
-  @Test(
-      dataProviderClass = TrackingOptInDataProvider.class,
-      dataProvider = "adsPetametricsDataProvider",
-      groups = "AdsOptInAcceptedOasis"
-  )
-  public void adsTrackingAcceptedForPetametrics(List<String> urlPatterns) {
-    networkTrafficInterceptor.startIntercepting();
-    TrackingOptInModal modal = new TrackingOptInModal();
-    modal.acceptOptInModal(driver, POLAND, ADS_HOME_PAGE);
-
-    modal.verifyTrackingRequestsSend(urlPatterns, networkTrafficInterceptor);
-  }
+//  @NetworkTrafficDump(useMITM = true)
+//  @Execute(trackingOptIn = false)
+//  @Test(
+//      dataProviderClass = TrackingOptInDataProvider.class,
+//      dataProvider = "adsNpaHiViDataProviderOasis",
+//      groups = "AdsOptInAcceptedOasis"
+//  )
+//  public void adsTrackingAcceptedForHiViAdSlots(List<String> urlPatterns) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)